packages/oscal-react-library/src/components/OSCALLoader.js

```diff
@@ -81,16 +82,16 @@ export default function OSCALLoader(props) {
     fetch(newOscalUrl)
       .then((response) => {
         if (!response.ok) throw new Error(response.status);
-        else return response.json();
+        else return response.text();
       }, handleError)
       .then((result) => {
-        if (!unmounted.current) {
+        if (!unmounted.current && result) {
+          const oscalObj = Convert.toOscal(result);
```

Comment:
This is probably the most significant change in this PR. Functionally, it ensures that we are validating every input document as _structurally_ valid OSCAL upon loading it.

This should be phenomenal for safety across the code base, even in places where we're still using JavaScript. The downside here is that we may reject documents that are "mostly" correct.

Note the emphasis on _structural_. Like all the TypeScript changes, this is focused on purely the structure of the objects (do fields exist) not the constraints (must meet particular string formats).

This may resolve #488.